published/005-2016-Feb-1.md

```diff
@@ -0,0 +1,209 @@
+  - [@lgierth](//github.com/lgierth) posted a writeup on the [migration from 0.3.x to 0.4.0](https://ipfs.io/blog/9-v04x-migration/).
+* [**specs**](//github.com/ipfs/specs/pull/37) The IPLD spec was merged, after four months. This reflected a ton of work from [@mildred](//github.com/mildred), [@findkiko](//github.com/findkiko), and [@jbenet](//github.com/jbenet). The [IPLD spec](https://github.com/ipfs/specs/blob/master/merkledag/ipld.md), the "thin-waist" merkle dag format, defines merkle-links, -dags, and -paths, as well as the IPLD Data model and formats. In short: JSON documents with named merkle-links that can be traversed.
+* [**distributions**](//github.com/ipfs/distributions)
+  - [dist.ipfs.io](http://dist.ipfs.io/) _almost_ has [signed releases](https://github.com/ipfs/distributions/pull/51) thanks to [@dignifiedquire](//github.com/dignifiedquire). This will be coming soon.
+	- [@noffle](//github.com/noffle) [worked](https://github.com/ipfs/distributions/pull/42) to get [ipget](http://ipget.me/) ready to be on [dist.ipfs.io](http://dist.ipfs.io/)
+* [**gx**](//github.com/whyrusleeping) 
+	- [@whyrusleeping](//github.com/whyrusleeping) shipped the GX PR, which pulls relevant code out of go-ipfs and into the new GX package manager. He also wrote a tool to publish IPNS records without the IPFS daemon.
+	- [@RichardLitt](//github.com/RichardLitt) spruced up the [gx README](https://github.com/whyrusleeping/gx) - it is almost ready to ship. 
+* [**js-ipfs**](//github.com/ipfs/js-ipfs) 
+	- Thanks to @diasdavid, `object features` now works, with tests and all. 
+	- As well, thanks to [@dignifiedquire](//github.com/dignifiedquire)'s efforts, the js-ipfs API now returns promises if there isn't a specified callback, allowing both major methods used by the Javascript community to work equally well.
+* [**registry-mirror**](//github.com/diasdavid/registry-mirror) 
+	- [@diasdavid](//github.com/diasdavid) merged registry-static into registry-mirror and removed all of the unnecessary stuff [(this was huge)](https://github.com/diasdavid/registry-mirror/pull/33).
+* [**station**](//github.com/ipfs/station)
+  - [@dignifiedquire](//github.com/dignifiedquire) fixed drag and drop
+    file uploading, and some dependency issues. So go ahead and try it
+    out.
+* [**ipfs-geoip**](//github.com/ipfs/ipfs-geoip)
+  - [@dignifiedquire](//github.com/dignifiedquire) rewrote the generation
+    script and cleaned up the code so that now the data is consistently
+    reproducable and fully stored on IPFS. This ensures that geoip lookups
+    over IFPS will work in all future releases.
+* [**fs-repo-migrations**](//github.com/ipfs/fs-repo-migrations)
+  - [@chriscool](//github.com/chriscool) Improved fs-repo-migrations tests by checking that the objects are the same and checking that garbage collection does not change anything. He also cleaned up sharness tests.
```

Comment:
this sounds exhaustively odd. how about:

- @chriscool improved important tests for fs-repo-migrations -- tests verify more edge cases when migrating forward and backward, through various sample workloads.